dropwizard-request-logging/src/main/resources/META-INF.services/io.dropwizard.request.logging.RequestLogFactory

```diff
@@ -1 +1,2 @@
 io.dropwizard.request.logging.LogbackAccessRequestLogFactory
+io.dropwizard.request.logging.old.LogbackClassicRequestLogFactory
```

Comment:
Should this file be located at `/META-INF/services/` instead of `/META-INF.services/`?